packages/dates/README.md

```diff
@@ -13,88 +13,160 @@ $ yarn add @shopify/dates
 
 ## Usage
 
-This library exports helpers, which allow React apps to easily work with dates and timezones.
+This library exports helpers that allow apps to easily work with dates and timezones.
 
-### apply-time-zone-offset
+### `applyTimeZoneOffset`
+
+Takes in a date string and two optional time zone strings parameters. Returns a new date string with the offset between the time zones added to it.
 
 ```ts
 import {applyTimeZoneOffset} from '@shopify/dates';
 
-const date = new Date('2018-06-01T14:00:00+00:00');
-const timezone = 'Australia/Perth';
+const date = new Date('2018-06-01Z14:00');
+const timeZone1 = 'Australia/Perth';
+const timeZone2 = 'America/Toronto';
 
-const month = getDateTimeParts(date, timeZone).month();
-const year = getDateTimeParts(date, timeZone).year();
+const newDate = applyTimeZoneOffset(date, timeZone1, timeZone2);
 ```
 
-### get-date-time-parts
+### `getDateTimeParts`
+
+Takes in a date string and an optional time zone parameter. Returns an object with functions to get the year, month, day, weekday, hour, minute and second of the provided date string.
```

Comment:
This also takes an object, not a string.

Reply:
Most of these take `Date` objects, not strings. Might want to double check all the types :)